docs/extensions.md

```diff
@@ -77,41 +77,50 @@ Open the Certificate file in a text editor, and copy it's contents in the field.
 
 After adding them, click "Save", saving the configuration. After saving, log out of your user to verify the SSO configuration. If you don't see a button for "Use SSO", you most likely configured the wrong organization.
 
-### Keycloak
-* Go create a realm in your keycloak by clicking the add realm button
-
-![31 01 2022_19 01 10_REC](https://user-images.githubusercontent.com/31187099/151965784-1798e815-1e57-4720-a875-ba379610a38e.png)
-
-* Enter a name for your realm
-* Click create and your realm is created
-* Under configure, move down to identity providers, on the pop up, click on the drop down select SAML v2.0
-
-* An add identity provider window pops up where you can configure your identity provider and SAML configs.
-* Here you will fill the:
-    - Alias field
-    - Display name
-    - Ensure Enabled field is turned on
-    - Ensure Trust Email field is turned on
-    
-![01 02 2022_11 39 39_REC](https://user-images.githubusercontent.com/31187099/151965938-3bddb6e8-6819-41dc-b6ae-85cd0b56b31e.png)
-
-* Still on this window scroll down to SAML Config and click on this to drop down the settings
-
-![01 02 2022_11 45 52_REC](https://user-images.githubusercontent.com/31187099/151966312-33b75cec-3906-4485-8c58-26d77dd2ef84.png)
-
-* Here you will fill the:
-    - Service Provider Entity ID - Scroll up to the Redirect URI and copy everything BEFORE /broker/(your-alias)/endpoint and paste it in this field.
-    - Single Sign-On Service URL - In this field paste https://<URL>:<PORT>/api/v1/login_sso 
-    - Scroll down and ensure Allow create is on 
-    - Ensure HTTP-POST Binding Response is on
-    - Ensure HTTP-POST Binding for AuthnRequest is on
-    - Ensure Validate Signature is on, paste your X509 certificate see [here](https://lightbend.github.io/ssl-config/CertificateGeneration.html)
-
-* Proceed to hit save
-
-* Copy the service Provider Entity ID and paste it in your Shuffle instance under the SSO Entrypoint (IdP) proceed to add /account at the end of your URI
-
-![01 02 2022_14 59 32_REC](https://user-images.githubusercontent.com/31187099/151966347-d56f78d5-9a04-4719-b55e-4e5d3ed6e703.png)
+### Open ID SSO setup with Keycloak
+In Keycloak 
+- Click on clients 
+- Create new client
+- Set client id 
+- Ensure client protocol is openid-connect
+- Hit save
+
+You now have a client. Click on settings and configure them as follows
+-Your client ID
+-Set up a name
+-Client Protocol set to openid-connect
+-Access Type set to public
+-Standard Flow Enabled toggled to ON
+-Direct access grants toggled to ON 
+
+https://user-images.githubusercontent.com/31187099/162692070-ccc54692-6793-4331-adc7-2356d0fd5397.jpg
```

Comment:
Can you make this and the other images an image that is rendered?

It's done like this:
![some text about the image](the actual image)

Markdown docs: https://www.markdownguide.org/basic-syntax/#images

Reply:
Done.